build-logic/src/main/kotlin/dokkabuild/utils/gradleKotlinCompatibility.kt

```diff
@@ -20,7 +20,7 @@ fun Project.configureGradleKotlinCompatibility() {
      * To be compatible with the AA, Dokka analysis should be compiled with approximately the same version
      * See https://kotlinlang.org/docs/kotlin-evolution-principles.html#evolving-the-binary-format
      */
-    val analysisK2Projects = listOf("analysis-kotlin-symbols")
+    val analysisK2Projects = listOf("analysis-kotlin-symbols", "runner-maven-plugin")
```

Comment:
As the `runner-maven-plugin` is an included build, it's possible to safely remove flags [there](https://github.com/Kotlin/dokka/blob/e56edc64823a651d278740e8ef9b74820555cc9a/dokka-runners/runner-maven-plugin/gradle.properties#L9-L10) in `gradle.properties`.